server-profiles/11.2.2/instance/bin/run.properties.subst

```diff
@@ -0,0 +1,561 @@
+# PingFederate Initialization Properties File
```

Comment:
Do we need these entire properties files? Could we remove them, or just set a few necessary properties via environment variables?

Reply:
added some env var substitution for the ones available. This is now in one file too, so not much maintenance required and it shouldn't change often (or at all)